### ISSUE TITLE
Is there a name or any prior art for this idea?

### ISSUE THREAD

**Issue by danielbachhuber** [nayafia/lemonade-stand#14]
Apologies if this isn't the right venue, but I thought it would be as good of a place as any for the conversation.

I'm starting a new business called [runcommand](https://runcommand.io/) for WordPress-based businesses to purchase enterprise support for [WP-CLI](http://wp-cli.org/). But it's a support with a twist: as a part of their plan, they also get to suggest ideas for new commands, or prioritize development of features against existing commands. The exact mechanics of how this will work is TBD, but the gist of the idea is that they'll get some number of tokens to put toward a given development task, and once the development task has enough tokens it will proceed forward.

I feel confident there's enough overlap between the companies in my target audience that they could greatly benefit from ongoing improvements to common infrastructure if they could somehow share the maintenance burden. Right now, the cost of sharing infrastructure is high — a company has to shoulder the burden of maintaining an entire project. runcommand permits companies to "rent" maintenance burden in a sense.

Can you think of a name or any prior art for this idea? I'd like to think I could just copy someone else's existing model, instead of needing to invent all of the project management / development workflow from scratch.

**Comment by pramsey** [comment]
That's basically the crowd-funding-one-time model, but run over and over and over. It has all the same problems, unfortunately. My experience w/ that model is that a surprisingly large proportion of the user community is content to just sit back and wait to see what chance will bring in the next release. Proactively funding features is pretty rare, and it's also dismaying how rarely the "urgent! I'll spend money for this feature" feeling occurs simultaneously (or near enough) in two funders. Inevitably, when the funded work is released, a pile of folks go "wow, yay, how awesome!" but that doesn't improve your income one bit.

**Comment by danielbachhuber** [comment]
For anyone coming across this thread, I've [posted an update to the runcommand blog](https://runcommand.io/2016/07/13/open-source-doing-it-wrong/)

tl;dr it's not easy — still refining the model :)
